### PR TITLE
docs: Complete README with more information

### PR DESCRIPTION
**Pull request by soulcramer**
## 📋 Changes description

Complete README with more information

## 🤔 Context

We need to add basic informations like what the project is, how to install it and how can someone contribute to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+<p align="center">
+<picture>
+    <source media="(prefers-color-scheme: dark)" srcset="art/spark-logo-dark.svg">
+    <img alt="Spark Design System logo" src="art/spark-logo-light.svg">
+  </picture>
+</p>
 
 [![👷 Build → 🧑‍🔬 Test → 🕵️ Lint](https://github.com/adevinta/spark-android/actions/workflows/ci.yml/badge.svg)](https://github.com/adevinta/spark-android/actions/workflows/ci.yml)
+
+Compose Spark Design System is based on Material 3 compose artifact described
```
Comment (SimonMarquis) [inline README.md:12]:
```suggestion
Compose Spark Design System is based on Material 3 Compose artifact described
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+on the [official documentation](https://material.io/) and maintained by Google developers
+and designers.
+
+But these native components and tokens are overridden to respect Spark's Visual Identity. You'll
+find
+the design specifications and technical information for supported platforms by Adevinta on
+[https://zeroheight.com/1186e1705/p/25ae4e-spark](https://zeroheight.com/1186e1705/p/25ae4e-spark).
```
Comment (SimonMarquis) [inline README.md:19]: nit:

```suggestion
[zeroheight](https://zeroheight.com/1186e1705/p/25ae4e-spark).
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+
+The demo app is not currently available available, but we plan to publish it in the near future.
```
Comment (SimonMarquis) [inline README.md:21]:
```suggestion
The demo app is not currently available, but we plan to publish it in the near future.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+
+## 🚀 Getting Started
+
+A `SparkTheme` is available from where you can get all
+colors, typographies and shapes in your composable hierarchy. Note that this theme is
+mandatory if you want to use any composable available.
```
Comment (SimonMarquis) [inline README.md:27]:
```suggestion
mandatory if you want to use any Spark composable.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,74 @@
...
+If you don't use it, an error will be triggered at the runtime.
```
Comment (SimonMarquis) [inline README.md:28]:
```suggestion
If you don't use it, an error will be thrown at runtime.
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,74 @@
...
+SparkTheme {
+    // Your composable declarations
+}
+```
+
+## Installation
+
+Just add the dependency to the `implementation` configuration.
+
+[//]: # ([![Maven Central]&#40;https://img.shields.io/maven-central/v/com.adevinta.spark/spark.svg&#41;]&#40;https://mvnrepository.com/artifact/com.adevinta.spark/spark&#41;)
```
Comment (SimonMarquis) [inline README.md:40]: This is not a valid markdown image/link

File: README.md [hunk 7]
```diff
@@ -1,3 +1,74 @@
...
+
+```kotlin
+SparkTheme {
+    // Your composable declarations
+}
+```
+
+## Installation
+
+Just add the dependency to the `implementation` configuration.
```
Comment (SimonMarquis) [inline README.md:38]: Developer are used to this, we don't need to be this "precise" IMO.

```suggestion
Add the main Spark dependency.
```

File: README.md [hunk 8]
```diff
@@ -1,3 +1,74 @@
...
+}
+```
+
+## Contributing
+
+Please see [the contribution guide](docs/CONTRIBUTING.md) and
+the [Code of conduct](docs/CODE_OF_CONDUCT.md) before contributing.
+
+License
+--------
```
Comment (SimonMarquis) [inline README.md:54]:
```suggestion
## License
```

Let's keep the same heading throughout this file

File: README.md [hunk 9]
```diff
@@ -1,3 +1,73 @@
...
+SparkTheme {
+    // Your composable declarations
+}
+```
+
+## Installation
+
+Add the main Spark dependency:
+
+[//]: # ([![Maven Central]&#40;https://img.shields.io/maven-central/v/com.adevinta.spark/spark.svg&#41;]&#40;https://mvnrepository.com/artifact/com.adevinta.spark/spark&#41;)
```
Comment (SimonMarquis) [inline README.md:40]:
```suggestion
Add the main Spark dependency: [![Maven Central](https://img.shields.io/maven-central/v/com.adevinta.spark/spark?label=%20&color=success)](https://search.maven.org/artifact/com.adevinta.spark/spark) [![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/com.adevinta.spark/spark?label=%20&color=lightgrey&server=https%3A%2F%2Fs01.oss.sonatype.org%2F)](https://s01.oss.sonatype.org/content/repositories/snapshots/com/adevinta/spark/spark/)
```

File: README.md (hunk 1 above)
Comment (SimonMarquis) [inline README.md]: Do we really want to name it `Compose Spark Design System`? and not just `Spark Design System` ?

```suggestion
Spark Design System is based on Material 3 compose artifact described
```

Comment (soulcramer) [reply README.md]: I've talked with iOS since they'll support UI Kit and SwiftUI and they won't specify the tech it's based on 👍

**Review by SimonMarquis — APPROVED** [review]

**Review by SimonMarquis — APPROVED** [review]

**Pull request merged**